src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1146,10 +1147,12 @@ protected override void GetItem(string path)
                         }
                     }
                 }
+#endif
 
                 FileSystemInfo result = GetFileSystemItem(path, ref isContainer, false);
                 if (result != null)
                 {
+#if !UNIX
                     // If we want to retrieve the file streams, retrieve them.
                     if (retrieveStreams)
```

Comment:
On Unix, a colon is completely allowed in a filename, so we want that to continue working as just a filename.  Tests were added to make sure this works.

I'll fix other cases of using the `file:stream` syntax.